src/sql/workbench/contrib/dataExplorer/browser/extensions.contribution.ts

```diff
@@ -53,7 +53,7 @@ MenuRegistry.appendMenuItem(MenuId.DataExplorerContext, {
 // Flat File Import
 MenuRegistry.appendMenuItem(MenuId.DataExplorerContext, {
```

Comment:
While you're here, probably should correct the ordering of Flat File Import vs Schema Compare (just move Schema Compare up one so their ordering in code reflects their order number.)